topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,364 @@
+"SRR998584.fastq.vcf" = "L5",
+"SRR13046689.fastq.vcf" = "bovis"
+)
+
+# Create tree with lineage as tip label instead of strain name
+
+tree_lineages <- tree_rooted
+tree_lineages$tip.label <- as.character(mtbc_lineages[tree_rooted$tip.label])
+
+# Define some colors for the lineages
+
+color_code_lineages = c(
+  L1 = "#ff00ff",
+  L2 = "#0000ff",
+  L3 = "#a000cc",
+  L4 = "#ff0000",
+  L5 = "#663200",
+  L6 = "#00cc33",
+  L7 = "#ede72e",
+  bovis="black",
+  orygis="black")
+
+pal_lineages <- as.character(color_code[tree_lineages$tip.label])
+
+par(mfrow = c(1, 2))
+plot(tree_rooted,cex = 0.7, root.edge = T)
+plot(tree_lineages,cex = 0.8, tip.color = pal_lineages, root.edge = TRUE)
+
+```
+
+<img src="./images/tree_rooted_lineages.svg" alt="drawing" width="600"/>
+
+
+### Questions
+> ### {% icon question %} Question 1
+>
+> Looking at the different lineages present in the tree, does our phylogeny make sense? Or asked differently: does our phylogeny show the same branching patterns between lineages as the established phylogeny in Fig. 1A?
+>
+> > ### {% icon solution %} Solution
+> >
+> > There is indeed a problem with our phylogeny: one L2 and one L5 strain do not cluster with the other strains of these lineages. Instead, they appear near the root of the tree, with very short (ERR5987300) to non-existent (ERR1203059) branches. Other parts of the tree are consistent with Fig. 1A, suggesting that we can focus our first round of trouble shooting on these two strains.
+> >
+> {: .solution}
+>
+{: .question}
+
+> ### {% icon question %} Question 2
+>
+> An important part of bioinformatics consists in trying to find out whether a surprizing observation has biological significance - or reflects a mistake somewhere in the numerous steps leading to the result. Have we just discovered two new lineages of MTB, or did we commit a stupid mistake? To find out, take a look a the TB-profiler and the VCF files for the two strange strains, and compare them with "normal" strains. Do you notice something?
+>
+> > ### {% icon solution %} Solution
+> >
+> > The VCF files hold the key to explain our puzzling observation: ERR1203059.vcf contains not a single SNP, ERR5987300.vcf only 81 SNPs. By contrast, the other strains have between 750 and 1250 SNPs. What happened here?
+> >
+> {: .solution}
+>
+{: .question}
+
+> ### {% icon question %} Question 3
+>
+> Recall the clusters identified in the previous tutorial, reproduced below. How do these clusters show up in the phylogenetic tree? What additional information does the tree contain?
+>
+| Sample       | Cluster_id | DR profile | Clustering  |
+|--------------|------------|------------|-------------|
+| ERR5987352   | 10         | Pre-MDR    | Clustered   |
+| ERR6362484   | 10         | Pre-MDR    | Clustered   |
+| ERR6362138   | 12         | MDR        | Clustered   |
+| ERR6362156   | 12         | Pre-XDR    | Clustered   |
+| ERR6362253   | 12         | MDR        | Clustered   |
```

Comment:
this table seems to be lacking the blockquote indentation, which will break the box

Reply:
blockquotes added